api/controllers/CommunityController.js

```diff
@@ -11,7 +11,7 @@ const welcomeMessage = 'Thank you for joining us here at Hylo. ' +
 
 module.exports = {
   find: function (req, res) {
-    Community
+    return Community
```

Comment:
why did you modify Community.find? as far as i can tell it's no longer used by the front-end